src/Solr/Implementor/ProductRepository.php

```diff
@@ -23,9 +23,11 @@
     public function getProductsInChunks($storeId, ProductIdChunks $chunks);
 
     /**
+     * @param null|int $sliceId
+     * @param null|int $totalNumberSlices
      * @return int[]
      */
-    public function getAllProductIds();
+    public function getAllProductIds($sliceId = null, $totalNumberSlices = null);
```

Comment:
Note that changes in interfaces implemented by other packages are a BC break and require a major version bump. So the next release of `solr-base` must be 2.0.0